tests/aodvv2/aodv_writer_tests.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2015
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       tests for the AODVv2 writer
+ *
+ * @author
+ *
+ * @}
```

Comment:
I think this closing tag must be at the far end of the header behind the `#endif` header guard